webapp/src/main/webapp/templates/freemarker/body/captcha/captcha-clientExecutionLogic.ftl

```diff
@@ -0,0 +1,39 @@
+<#assign isEnabled = isEnabled!true>
+
+${stylesheets.add('<link rel="stylesheet" href="${urls.base}/templates/freemarker/edit/forms/css/customForm.css" />')}
+${scripts.add('<script type="text/javascript" src="${urls.base}/js/commentForm.js"></script>',
+              '<script type="text/javascript" src="${urls.base}/js/jquery-ui/js/jquery-ui-1.12.1.min.js"></script>')}
+
+<#if isEnabled && captchaToUse == "RECAPTCHAV2">
+    <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+    <script>
+        document.getElementById('contact_form').addEventListener('submit', function() {
+            var recaptchaResponse = grecaptcha.getResponse();
+            console.log(recaptchaResponse);
```

Comment:
Is it intentional to leave this log?

Reply:
No, I forgot to remove it, thank you for noticing!